### DISCUSSION TITLE
[Script request] Storyteller

### DISCUSSION THREAD

**Discussion by mikemathewson** [community-scripts/ProxmoxVE#833] [Request script]
### Application Name

Storyteller

### Website

https://smoores.gitlab.io/storyteller/

### Description

Storyteller is a self-hosted platform for creating and reading ebooks with synced narration. It's made of of two components: the synchronization server, and the mobile apps. Together, these components allow you to take audiobooks and ebooks that you already own and automatically synchronize them, as well as read or listen to (or both!) the resulting synced books.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by jobocodetho** [comment]
+1

Appears to have moved: [https://storyteller-platform.gitlab.io/storyteller/](https://storyteller-platform.gitlab.io/storyteller/)

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1617
